patches/ios-chrome-app-BUILD.gn.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
+index 52fe2a5587fd536c0a250ffc00ca0d21ebada838..8c0e7581cce0acd15e2d03db2e8620954b981e58 100644
+--- a/ios/chrome/app/BUILD.gn
++++ b/ios/chrome/app/BUILD.gn
+@@ -754,8 +754,7 @@ ios_app_bundle("chrome") {
+   # (see https://crbug.com/1394553 for details of the build failure). This
+   # is a stop-gap solution.
+   if (target_environment == "catalyst" ||
+-      ios_provider_target !=
```

Comment:
is this an upstream change we're uplifting?

Reply:
I’ve submitted a fix for this upstream, yes, so we should revisit this in the next version.

Reply:
mostly I'm asking because normally we would adjust this patch to change fewer lines, but if this is a mistake that upstream will fix then this is ok

Reply:
I mean, I fixed it myself upstream, but yeah.

Reply:
haha, well I just meant something they are going to merge :)